src/http.rs

```diff
@@ -106,7 +140,14 @@ fn appdata(
 			extrinsics: data,
 		}),
 
-		Ok(None) => ClientResponse::NotFound,
+		Ok(None) => {
+			let lock = counter.lock().unwrap();
```

Comment:
Same...

Reply:
changing the function to take u64 instead of Arc<>